superset-frontend/src/components/FilterableTable/FilterableTable.tsx

```diff
@@ -152,7 +165,7 @@ export default class FilterableTable extends PureComponent {
     this.fitTableToWidthIfNeeded();
   }
 
-  getDatum(list, index) {
+  getDatum(list: List<Record<string, CellDataType>>, index: number) {
```

Comment:
Can create a `Datum` type for `Record<string, CellDataType>`. I counted 11 places that used it.

Reply:
yeah, i was going back and forth on this since Record<string, CellDataType> is a bit more descriptive than Datum. But I guess IDEs will expand it anyway, so it's better to adhere to DRY. Will update